### ISSUE TITLE
"IndexError: list index out of range" when looking for ep_kanji of specific episode

### ISSUE THREAD

**Issue by msdos** [adameste/anidbcli#9]
Traceback:

```
[INFO] Processing file "[Bloat]_Neon_Genesis_Evangelion_-_12_[290900C5].mkv"
[SUCCESS] Generated ed2k link.
Traceback (most recent call last):
  File "/usr/local/bin/anidbcli", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/anidbcli/cli.py", line 111, in main
    cli(obj={})
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/anidbcli/cli.py", line 82, in api
    res = operation.Process(file_obj)
  File "/usr/local/lib/python3.5/dist-packages/anidbcli/operations.py", line 110, in Process
    fileinfo["ep_kanji"] = parsed[39]
IndexError: list index out of range
```

 `parsed` contents before the exception (length of 39):

```python
['914683', '22', '124', '10303', '0', '4097', '554862092', 'f039ea49b97f2f2f64263d23adbf7951', '36bd827a5d2f5a525015e78ced17aac0', '9fe7e2dd8fcc3ea4b7d8845291e7f7dfcbab3828', '290900c5', '', 'high', 'DVD', 'AC3', '448', 'H264|AVC', '2700', '704x480', 'mkv', 'japanese', 'english', '1401', '819417600', 'Shinseiki Evangelion - 12 - She Said: "Don\'t Make Others Suffer for Your Personal Hatred." - [Bloat](290900c5).mkv', '26', '26', '1995-1996', 'TV Series', 'Japan,Piloted Robot,Shounen,Angst,New,Mecha,Twisted,Violence,Action,Genetic Modification,Alien,Middle School,Post-apocalypse,Gunfights,Sci-Fi,Air Force,Military,Nopan,Conspiracy,Religion,Future,Ecchi,Small Breasts,Earth,Americas,Germany,Europe,United States of America,Pantsu,Navy,Running with Food in Mouth,Nudity', 'Shinseiki Evangelion', '新世紀エヴァンゲリオン', 'Neon Genesis Evangelion', "新世紀エヴァンゲリオン'新世纪福音战士'Neon Genesis Evangelion'Евангелион'신세기 에반게리온'ניאון ג'נסיס אוונג'ליון'อีวานเกเลี่ยน: มหาสงครามวันพิพากษา'Evangelion", "nge tv'NGE'ева'eva'sse'eva tv'єва", "Evangelion TV'Neon Genesis Evangelion Renewal'Neon Genesis Evangelion Platinum'Евангелион'Evangelion'Євангеліон нового покоління'Нова ера: Євангеліон'Євангеліон", '12', 'She Said: "Don\'t Make Others Suffer for Your Personal Hatred."', 'Kiseki no Kachi w']
```

`parsed` contents of an episode (E11 - The Day Tokyo-3 Stood Still.mkv) from the same series that is ok (length 42):

```python
['914682', '22', '123', '10303', '0', '4097', '517199348', 'bf5088b9e394ee35c7599846c75f2f1c', '1ff57b689f08667e8920065b3ad047ea', '83a1b4628f1e3d047532dd5b02bc214e4ff1667f', '1c50d9d5', '', 'high', 'DVD', 'AC3', '448', 'H264|AVC', '2485', '704x480', 'mkv', 'japanese', 'english', '1401', '818812800', 'Shinseiki Evangelion - 11 - The Day Tokyo-3 Stood Still - [Bloat](1c50d9d5).mkv', '26', '26', '1995-1996', 'TV Series', 'Japan,Piloted Robot,Shounen,Angst,New,Mecha,Twisted,Violence,Action,Genetic Modification,Alien,Middle School,Post-apocalypse,Gunfights,Sci-Fi,Air Force,Military,Nopan,Conspiracy,Religion,Future,Ecchi,Small Breasts,Earth,Americas,Germany,Europe,United States of America,Pantsu,Navy,Running with Food in Mouth,Nudity', 'Shinseiki Evangelion', '新世紀エヴァンゲリオン', 'Neon Genesis Evangelion', "新世紀エヴァンゲリオン'新世纪福音战士'Neon Genesis Evangelion'Евангелион'신세기 에반게리온'ניאון ג'נסיס אוונג'ליון'อีวานเกเลี่ยน: มหาสงครามวันพิพากษา'Evangelion", "nge tv'NGE'ева'eva'sse'eva tv'єва", "Evangelion TV'Neon Genesis Evangelion Renewal'Neon Genesis Evangelion Platinum'Евангелион'Evangelion'Євангеліон нового покоління'Нова ера: Євангеліон'Євангеліон", '11', 'The Day Tokyo-3 Stood Still', 'Seishi Shita Yami no Naka de', '静止した闇の中で', 'Bloat', 'Bloat']
```

I'm not familiar to what's supposed to be returned from the API, so I'm just providing the error and details. Any more information or print that you would need just ask me.

**Comment by adameste** [comment]
Will look into it

**Comment by adameste** [comment]
Should be resolved in version 1.40. FIle info is probably too long to fit into single packets, so i tried splitting info grabbing into 2. Try it and share your result :)

**Comment by msdos** [comment]
1.40 fixed the problem for this episode. Thanks!
